dev-libs/libdigidocpp/libdigidocpp-3.14.4.ebuild

```
@@ -44,15 +44,15 @@ src_prepare() {
 	if ! has_version app-editors/vim-core; then
 		epatch "${FILESDIR}/xxdi.patch"
```

`epatch` is no longer allowed in EAPI 7, and `eapply` should be used instead. In this case, it fails silently, which makes the compilation fail on my system (that doesn't have `vim` installed). This change has fixed it for me:
```suggestion
		eapply "${FILESDIR}/xxdi.patch"
```